riscv_isa.cpp

```diff
@@ -1405,4 +1405,58 @@ void ac_behavior(FMV_D){
   float temp;
   temp = load_float(rs1);
   save_float(temp, rd);
-}
\ No newline at end of file
+}
+
+
+// Instruction FEQ_D behavior method
+void ac_behavior(FEQ_D){
+  dbg_printf("FEQ.D r%d, r%d, r%d \n", rd, rs1, rs2);
+  dbg_printf("RBF[rs1] = %f \n", load_double(rs1));
+  dbg_printf("RBF[rs2] = %f \n", load_double(rs2));
+  if((isnan(load_double(rs1)) == 1) || (isnan(load_double(rs2)) == 1))
```

Comment:
This "isnan" call is failing to compile if I generate the model with the "-gdb" flag in acsim:
riscv_isa.cpp:1425:29: error: call of overloaded 'isnan(double)' is ambiguous

Replace this with a custom "isnan", if you need isnan. See https://github.com/ArchC/mips/blob/mips32r2/mips_isa_helper.H#L58-L60